lib/smart_answer/logged_error.rb

```diff
@@ -1,14 +1,10 @@
 module SmartAnswer
   class LoggedError < BaseStateTransitionError
+    attr_reader :log_message
```

Comment:
is this line really required ?

Reply:
@ikennaokpala it seems it raises an Error otherwise:
```
Error:
FlowTest#test_: sequence of two questions a question raises a logged error should notify Sentry. :
NoMethodError: undefined method `log_message' for #<SmartAnswer::LoggedError: Sorry, that's not valid>
    lib/smart_answer/flow.rb:127:in `rescue in block in process'
    lib/smart_answer/flow.rb:122:in `block in process'
    lib/smart_answer/flow.rb:120:in `each'
    lib/smart_answer/flow.rb:120:in `inject'
    lib/smart_answer/flow.rb:120:in `process'
    test/unit/flow_test.rb:292:in `block (3 levels) in <class:FlowTest>'
    test/unit/flow_test.rb:303:in `instance_exec'
    test/unit/flow_test.rb:303:in `block in create_test_from_should_hash'

bin/rails test test/unit/flow_test.rb:290
```